ydb/core/fq/libs/checkpoint_storage/ydb_state_storage.cpp

```diff
@@ -240,33 +399,94 @@ TFuture<TIssues> TStateStorage::Init()
             ss << ", issues: ";
             issues.PrintTo(ss);
         }
-
-        //LOG_STREAMS_STORAGE_SERVICE_DEBUG(ss.Str());
     }
 
     return MakeFuture(std::move(issues));
 }
 
+EStateType TStateStorage::DeserealizeState(TContext::TaskInfo& taskInfo)
+{
+    TString blob;
+    for (auto it = taskInfo.Rows.begin(); it != taskInfo.Rows.end();) {
+        blob += *it;
+        it = taskInfo.Rows.erase(it);
+    }
+    taskInfo.States.push_front({});
+    NYql::NDqProto::TComputeActorState& state = taskInfo.States.front();
+    if (!state.ParseFromString(blob)) { // backward compatibility with YQL serialization
```

Comment:
Can we have the old blob that will successfully parse?

Reply:
I think no, Vasiliy confirmed that it is probably no longer used. I delete comment